docs/libssh2_channel_read_host_info.3

```diff
@@ -0,0 +1,36 @@
+.TH libssh2_channel_read_host_info 3 "23 Jan 2019" "libssh2 1.8.0" "libssh2 manual"
+.SH NAME
+libssh2_channel_read_host_info - read host information.
+.SH SYNOPSIS
+#include <libssh2.h>
+
+LIBSSH2_API int libssh2_channel_read_host_info(LIBSSH2_CHANNEL *channel,
```

Comment:
how about dropping `read` from the name and just make it `libssh2_channel_hostinfo` ? Is it even info about the *host* ?